crypto/dh_extra/dh_test.cc

```diff
@@ -851,8 +851,11 @@ TEST(DHTest, GenerateKeyTwice) {
   bssl::UniquePtr<BIGNUM> g(BN_new());
   ASSERT_TRUE(g);
   ASSERT_TRUE(BN_set_word(g.get(), 2));
-  bssl::UniquePtr<DH> key1 = NewDHGroup(p.get(), /*q=*/nullptr, g.get());
+  bssl::UniquePtr<DH> key1(DH_new());
```

Comment:
This commit can be skipped since we have the final version of this test already.